nuxt-app/package.json

```diff
@@ -5,15 +5,16 @@
     "build": "nuxt build",
     "dev": "nuxt dev",
     "generate": "nuxt generate",
+    "playwright:show-report": "npx playwright show-report ../playwright-reports/nuxt-app-playwright-report",
     "preview": "nuxt preview",
     "postinstall": "nuxt prepare",
-    "test:system": "APP_NAME=nuxt-app npx playwright test -c ../playwright.config.ts --project=system:nuxt",
+    "test:system": "APP_NAME=nuxt-app npx playwright test -c ../playwright.config.ts --project=system",
```

Comment:
Is `--project=system` required if there is only one entry in the projects array?